src/sql/workbench/contrib/notebook/browser/notebook.contribution.ts

```diff
@@ -252,7 +252,12 @@ configurationRegistry.registerConfiguration({
 			'default': 1.5,
 			'minimum': 1,
 			'description': localize('notebook.markdownPreviewLineHeight', "Controls the line height used in the notebook markdown preview. This number is relative to the font size.")
-		}
+		},
+		'notebook.showRenderedNotebookInDiffEditor': {
```

Comment:
Any thoughts about making this default true? Or is it going to be fine to have the customers who were asking for it to have to go in and enable it while you're still working on the feature?

Reply:
The subset of customers who've said that they need this specific scenario all work in the same workspace (and this setting can be enabled globally there). I'm a little hesitant to turn it on for other users, since the line-by-line diff isn't there, so I wouldn't want folks to think that there's a regression in functionality.